README.md

```diff
@@ -215,14 +217,76 @@ Endpoint  | Description
 `/health` | healthcheck - returns `OK`
 `/events` | event sink - returns `OK` if all keys are set in an event, error message otherwise
 
-### Known limitations
+## Advanced usage
 
-The following section describes known limitations in `marathon-consul`.
+If you need to map your Marathon task into multiple service registrations in Consul, you can configure marathon-consul 
+via Marathon's `portDefinitions`:
 
-* In Marathon when a deployment changing the application's service name (by changing its `labels`) is being stopped, it changes app's configuration anyway.
-  This means we loose the link between the app and the services registered with the old name in Consul.
-  Later on, if another deployment takes place, new services are registered with a new name, the old ones are not being deregistered though.
-  A scheduled sync is required to wipe them out.
+```
+  "id": "my-new-app",
+  "labels": {
+    "consul": "true",
```

Comment:
I opt for using empty name. I think it's time to cleanup `consul: true` logic but not in this PR.